packages/dd-trace/src/plugins/util/web.js

```diff
@@ -8,6 +8,7 @@ const tags = require('../../../../../ext/tags')
 const types = require('../../../../../ext/types')
 const kinds = require('../../../../../ext/kinds')
 const urlFilter = require('./urlfilter')
+const { INCOMING_HTTP_REQUEST_START, INCOMING_HTTP_REQUEST_END } = require('../../gateway/channels')
```

Comment:
Events from plugin should be generic and not come from AppSec.

Reply:
This doesn't come from appsec, I've built the gateway to have exactly zero reference to appsec. This is just how we decided to name those two lifecycle events.

Reply:
I will use the http plugin's channels directly once the plugin gets converted, and i will remove these two channels